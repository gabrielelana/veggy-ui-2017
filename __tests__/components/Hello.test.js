import React from 'react'
import renderer from 'react-test-renderer'
import Hello from '../../src/app/Hello'

describe('<Hello />', () => {
  test('should render', () => {
    const component = renderer.create(<Hello />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})