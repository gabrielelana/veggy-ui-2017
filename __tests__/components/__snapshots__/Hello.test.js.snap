// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello /> should render 1`] = `
<div>
  Hello!
</div>
`;
